logicrunner/executionbroker.go

```diff
@@ -719,6 +727,10 @@ func (q *ExecutionBroker) AddRequestsFromLedger(ctx context.Context, transcripts
 	q.Prepend(ctx, true, transcripts...)
 }
 
+func (q *ExecutionBroker) EnqueueAbandonedOutgoingRequest(ctx context.Context, reqRef insolar.Reference, req *record.OutgoingRequest) {
+	q.outgoingSender.SendAbandonedOutgoingRequest(ctx, reqRef, req)
```

Comment:
Injection of dependency into execution broker, just to delegate. This is wrong. I think you should make outgoingSender a dependency of requestsFetcher and use it directly. My point is that you don't need EnqueueAbandonedOutgoingRequest on execution broker interface.

Reply:
ok fixed